app/models/plant_accession.rb

```diff
@@ -41,7 +56,7 @@ def self.table_columns
     [
       'plant_accession',
       'plant_lines.plant_line_name',
-      'plant_varieties.plant_variety_name',
+      'CASE WHEN plant_varieties.id IS NULL THEN plant_line_varieties.plant_variety_name ELSE plant_varieties.plant_variety_name END AS plant_variety_name',
```

Comment:
I know, and model/plant_accession_spec.rb should test that case (the example in line 16).